_events/2022/2022-06-community-call.md

```diff
@@ -0,0 +1,25 @@
+---
+title: June 2022 Community Call
+expires: 2022-06-11
+event_date: "June 10, 2022"
+layout: event
+duration: 60
+repeated: false
+category: community-call
+time:
+    - - start: 2022-06-10T18:00:00Z
+        end: 2022-06-10T19:00:00Z
+---
+
+# Whitepaper: State of RSE in the US
+
+The next community call will be on June 10, 2022 at 14ET/13CT/12MT/10PT.
+
+The US-RSE monthly community calls are experimenting with a new time to try and better accommodate Alaska and Hawaii time zones. The calls are moving to the **second Friday of each month**. 14:00-15:00 Eastern starting June 2022. Community call topics, and agenda and zoom registration announcements are posted to Slack and sent to US-RSE email lists.
+
+The June 2022 (on **Friday June 10th, 14:00-15:00 US Eastern time**) call will focus on starting a whitepaper on the State of RSE in the US. The call will kickoff with a quick review of some of the recent community calls that touched on this topic. In breakout groups we will try and experiment with outlining a set of key sub-topics and the structure for a whitepaper. Look forward to seeing those of you who can make on the Friday!
+
+Community calls typically have a topical focus. They provide a forum for US-RSE members to compare experiences around a topic, hang-out and chat on zoom. You can find the recordings of our monthly calls [here](https://www.youtube.com/playlist?app=desktop&list=PLMgG8jy8w-8neP6gPOgifH6kh4t_RUTTH).
```

Comment:
Did this change? I thought we weren't posting community call recordings publicly? Just on slack (and maybe email?)?

Reply:
Ups... I copied it from the newsletter and forgot to remove it. I'll delete it. It probably stays in the commits but I don't think they are findable through the search?